components/branding/ColorBlock.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Card, CardContent, Stack } from '@mui/material/'
+import { InfoBlock } from './InfoBlock'
+
+const hexToRgb = (hex) => {
```

Comment:
this doesn't need to live inside this component that gets recreated multiple times. let's pull this function out into our utils dir.